### ISSUE TITLE
Update Homepage Hero Image

### ISSUE THREAD

**Issue by GTBarrows** [codeforboston/boston-liquor-license-tracker#435]
The current Hero Image on the homepage is one of the most used stock images when it comes to bars on the internet, and I feel poorly reflects the goal of this site in facilitating Bars/Restaurants obtaining liquor licenses. There are also accessibility concerns regarding the contrast of the Hero text against the current Image.

As such the Hero Image is getting replaced with a more accessible and stylistically fitting image:

## New Image
<img width="4631" height="3242" alt="Image" src="https://github.com/user-attachments/assets/db4a72da-4c62-4306-bfd1-39631a1edaaf" />

> Attribution Info:
> Photographer: Louis Hansel
> Site: Unsplash
[>Link](https://unsplash.com/photos/person-pouring-liquor-in-clear-drinking-glass-yLUvnCFI500)

### Desktop
<img width="1371" height="781" alt="Image" src="https://github.com/user-attachments/assets/d23ec9fe-8f5b-48d5-8d67-d0d7b5937579" />

### Mobile
<img width="497" height="587" alt="Image" src="https://github.com/user-attachments/assets/4158258a-98f9-4b27-a261-ff4f8542ec4e" />

## Design Notes
To better improve visibility the image itself is offset to the right on the screen and then a linear gradient of black is overlayed over the left part of the screen

The Images Dimensions are: 1592px x 1116px, with the image being offset by roughly 30% to the right

_Code taken from Figma_
```background: linear-gradient(90deg, #000 29.81%, rgba(0, 0, 0, 0.20) 63.46%, rgba(0, 0, 0, 0.00) 100%), url(<path-to-image>) lightgray 591.303px -315.317px / 82.844% 230.913% no-repeat;```
background: linear-gradient(90deg, #000 37.02%, rgba(0, 0, 0, 0.00) 87.98%);

## Examples
**Image Offset Desktop**
<img width="1500" height="451" alt="Image" src="https://github.com/user-attachments/assets/6416f60d-cf36-4048-834e-ed2e00f02abb" />
**Gradient Desktop**
<img width="1466" height="534" alt="Image" src="https://github.com/user-attachments/assets/33fc98c0-4f14-4731-9ff3-3f51fa05f1ba" />
**Image Offset Mobile**
<img width="1320" height="872" alt="Image" src="https://github.com/user-attachments/assets/8fbdae70-93e5-472c-ac03-713204d6af57" />
**Gradient Mobile**
<img width="1323" height="885" alt="Image" src="https://github.com/user-attachments/assets/0fed3167-9309-4c33-87b6-3dd33f871806" />
